### DISCUSSION TITLE
Update Multiselect maxItems() translation

### DISCUSSION THREAD

**Discussion by stefanfortuin** [filamentphp/filament#2438] [Help] [answered]
Is it possible to manually set the translation of errors for the multiselect component?

Since the rebuild to Choicesjs the translation for maxItems() has been missing.
With a Dutch locale it will still say "Only 1 values can be added"

**Comment by danharrin** [comment, marked as answer]
Yeah, we can switch to passing in the validation translation string for [`maxItemText()`](https://github.com/Choices-js/Choices#setup)

**Comment by stefanfortuin** [comment]
Good, i already saw an option in the docs for Choicesjs that it would be possible.
I wil be eagerly waiting for the pull request.

**Reply by danharrin** [reply]
Are you not able to make the PR yourself? It's not really a priority for me ATM

**Comment by danharrin** [comment]
This has now been fixed by #4160
